examples/atari/atari_ppo.py

```diff
@@ -184,37 +186,44 @@ def dist(p):
     # collector
     train_collector = Collector(policy, train_envs, buffer, exploration_noise=True)
     test_collector = Collector(policy, test_envs, exploration_noise=True)
+
     # log
-    log_name = 'ppo_icm' if args.icm_lr_scale > 0 else 'ppo'
-    log_path = os.path.join(args.logdir, args.task, log_name)
-    if args.logger == "tensorboard":
-        writer = SummaryWriter(log_path)
-        writer.add_text("args", str(args))
-        logger = TensorboardLogger(writer)
-    else:
+    now = datetime.datetime.now().strftime("%y%m%d-%H%M%S")
+    algo_name = "ppo_icm" if args.icm_lr_scale > 0 else "ppo"
+    log_name = os.path.join(args.task, algo_name, str(args.seed), now)
```

Comment:
Could you add the `algo_name` to `args` so that we can filter all experiments whose `algo_name="dqn"` or group by `algo_name`?

Reply:
https://wandb.ai/tianshou/atari.benchmark/runs/mh6zb1zf?workspace=user-tianshou